gemfiles/rouge_1.gemfile

```diff
@@ -0,0 +1,62 @@
+# This file was generated by Appraisal
```

Comment:
True, actually those gemfiles are only needed when we want to run all the tests. Since in our case we only wants to run `focus` spec we can remove them.